From 883dd75159fe7e5ce6458bbc1c290c0484b39c31 Mon Sep 17 00:00:00 2001
From: Som Tripathi <tripathisom142004@gmail.com>
Date: Sun, 19 Jul 2026 02:53:39 -0500
Subject: [PATCH 07593/15333] staging: rtl8723bs: remove extra blank lines in
 rtw_qos.h

Fix checkpatch.pl warnings by removing extra blank lines:

- after the license header, before the include guard
- inside the include guard, before the struct definition
- before the closing #endif

No functional change.

Signed-off-by: Som Tripathi <tripathisom142004@gmail.com>
Assisted-by: Claude:claude-sonnet-5
Link: https://patch.msgid.link/20260719075339.115500-1-somtri@iastate.edu
Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>
---
 drivers/staging/rtl8723bs/include/rtw_qos.h | 4 ----
 1 file changed, 4 deletions(-)

diff --git a/drivers/staging/rtl8723bs/include/rtw_qos.h b/drivers/staging/rtl8723bs/include/rtw_qos.h
index 1f28837f6c27..733bfacef0b9 100644
--- a/drivers/staging/rtl8723bs/include/rtw_qos.h
+++ b/drivers/staging/rtl8723bs/include/rtw_qos.h
@@ -5,15 +5,11 @@
  *
  ******************************************************************************/
 
-
 #ifndef _RTW_QOS_H_
 #define _RTW_QOS_H_
 
-
-
 struct	qos_priv {
 	unsigned int	  qos_option;	/* bit mask option: u-apsd, s-apsd, ts, block ack... */
 };
 
-
 #endif	/* _RTL871X_QOS_H_ */
-- 
2.55.0

